cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -179,6 +185,7 @@ if [[ "${ATDM_SYSTEM_NAME}" == "" ]] && [[ "${knownSystemNameInBuildName}" != ""
   assert_selected_system_matches_known_host_in_build_name || return
   assert_selected_system_matches_known_system_type_mathces || return
 fi
+>>>>>>> upstream/develop
```

Comment:
@jeforster, looks like you forgot to remove a conflict marker?  Did you test this?

Reply:
Wow you are on it.  I saw and fixed that like 20 seconds after I pushed.

Reply:
@jefroster, do you run `git diff` before you commit?  Do you run `git log -p` for commits after you create them?  This was a small patch to this file so a `git log -p -1` would have showed this error immediately.  Then you could have fixed this in the file and ran `git commit --amend` to fix this before you pushed.